docs/Reference/Pantheon-API-Methods.md

```diff
@@ -904,6 +904,77 @@ Returns the account balance of the specified address.
+
+**Parameters**
+
+`DATA` - 20-byte address of the account or contract.
+
+`ARRAY` - 32-byte storage values of the account or contract address as described in [eth_getStorageAt](#eth_getstorageat)
+
+`QUANTITY|TAG` - Integer representing a block number or one of the string tags `latest`, `earliest`, or `pending`, as described in [Block Parameter](../HowTo/Interact/Pantheon-APIs/Using-JSON-RPC-API.md#block-parameter).
+
+**Returns**
+
+`result`: *Object* - Account details:
+
+* `balance`:`Quantity` - Account balance
+* `codeHash`:`Data, 32-byte` - Hash of the account code
+* `nonce`:`Quantity` - Number of transactions sent from the account
+* `storageHash`:`Data, 32-byte` - SHA3 of the `storageRoot`
+* `accountProof`:`Array` - RLP-encoded merkle tree nodes, starting with the `stateRoot`
+* `storageProof`:`Array`- Storage entries. Each entry is an object that displays: 
+     * `key`:`Quantity` - Storage key
+     * `value`:`Quantity` - Storage value
+     * `proof`:`Array` - RLP-encoded merkle tree nodes, starting with the `storageHash`
+
+!!! example
+    ```bash tab="curl HTTP" 
+    curl -X POST --data '{"jsonrpc":"2.0","method": "eth_getStorageAt","params": ["0x‭{"jsonrpc":"2.0","method": "eth_getProof","params": [
+    "0a8156e7ee392d885d10eaa86afd0e323afdcd95", ["0x0000000000000000000000000000000000000000000000000000000000000347"], "latest"],"id": 1}' http://127.0.0.1:8545
+    ```
+         
+    ```bash tab="wscat WS"
+    {"jsonrpc":"2.0","method": "eth_getProof","params": [
+    "0a8156e7ee392d885d10eaa86afd0e323afdcd95", ["0x0000000000000000000000000000000000000000000000000000000000000347"], "latest"],"id": 1}
+    ```
+         
+    ```json tab="JSON result"
+    {
+      "jsonrpc": "2.0",
+      "id": 1,
+      "result": {
+        "accountProof": [
+          "0xf90211a02dfb324fc04fb727c346b50b269c1e0bb6ba65b86c1d094f0e3211089ee46e4fa0db4405e5146db2a6f9e9dce7a77536003807c6660514e4afd25701bafeb581f3a0ac7318dd07b398de1209275051780a8870c38dd78060d050c934c78fd25c52dba004c7f825824b93f15e8126bb60e2f51233a6c9cb0b0a5abbf86817c0ab7b8530a02211f2b43b9bb4fa432625f2729b57ca655c88c38ca942152c6b7dba249b1964a0193e6b06d8244e36ca047c588f92bb4043eb7518fab35384e23ea228ea8e868ca026e67bbadf0f47f4275b6158035985de014a89d283bbf866a48f27a80e0cd05ba0e7d3d4565eec37add8090c6272cab0f02e828bea1f2da33a2893d46e3232d204a0aa3e83e318eaf74f3af93d26931637bc37065404dc253ac75c1f46eb8ce63b4ca0f005c4660b88fecef39ba7e92bea2a036602822b8eacd37b36d9091db2801214a0bdec15dc4fb5bb3038c0c45e8fd100b26776725daa178f076db60c3ed116cd91a0dd626c85ded41cde21515f31cd0849c39a26543fd16f2741e911dce03c349b15a0a3e54c7c25292e348538cbdf1280e2f67e6972722218429a84033a733528dd72a0e098cf9becc3929933266b5b7beb569c628f359858a4f7d5c006668a68e5fb2ea0e21b659d3980a21e94fbffe883885a6066265f1f671e2a78e4e5b2d0f38bef9ba0dd9f122902b09e22e6b14b0981d4595360d716b8f801790b952f7840608d898380",
```

Comment:
Do we really need all of the proof bytes?  Could we just elide them like so?

```suggestion
          "0xf90211a0...8d898380"
```

Reply:
Yes, it looks a lot better after adding the ellipsis.